backend/src/apiserver/resource/resource_manager.go

```diff
@@ -414,6 +414,15 @@ func (r *ResourceManager) CreateRun(ctx context.Context, apiRun *api.Run) (*mode
 
 	// Assign the create at time.
 	runDetail.CreatedAtInSec = runAt
+
+	// Assign the scheduled at time
+	if !apiRun.ScheduledAt.AsTime().IsZero() {
```

Comment:
To help me understand: Isn't that the schedule time is later than creation time? `ScheduleAt` means argo workflow starts to execute this run.

Reply:
I agree it's rather confusing. From the proto definition/comments, it seems that the "created_at" time is when the run started running; the "scheduled_at" time, instead, is the time people doing the action of scheduling.

https://github.com/kubeflow/pipelines/blob/d05110e0b0c300879614b3bb28c06fee71470de6/backend/api/run.proto#L241-L248

Reply:
Thank you for the reference! Then it makes sense with the current code here.

Reply:
Yes I agree with the `Scheduled time` can help to clear the confusion here. Thank you Linqing!